src/ai/susi/server/api/cms/GetSkillsByAuthor.java

```diff
@@ -131,13 +131,39 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
                 return new ServiceResponse(result);
             }
 
+            result.put("author_avatar", getUserAvatar(author_email));
             result.put("author_skills", authorSkills);
             result.put("accepted",true);
             result.put("message","Successfully fetched all Skills created by author.");
             return new ServiceResponse(result);
         }
     }
 
+ // Method to get the avatar image of the user
+    public static String getUserAvatar(String email) {
+        ClientIdentity identity = new ClientIdentity(ClientIdentity.Type.email, email);
+        Accounting accounting = DAO.getAccounting(identity);
+        String userId = identity.getUuid();
+        String avatarUrl = "";
+        String avatarType = "";
+        JSONObject accountingObj = accounting.getJSON();
+        if (accountingObj.has("settings") &&
+            accountingObj.getJSONObject("settings").has("avatarType")) {
+            avatarType = accountingObj.getJSONObject("settings").getString("avatarType");
+        } else {
+            avatarType = "default";
+        }
+
+        if(avatarType.equals("gravatar")) {
```

Comment:
This is not the correct way. Check how getAvatar endpoint is implemented.

Reply:
@akshatnitd what is the correct way? Please advise @Dilshaad21 in greater detail to get this done correctly